examples/digital-identity/index.js

```diff
@@ -18,7 +18,7 @@ app.use('/static', express.static('static'));
 
 const router = express.Router();
 
-router.get('/createSession', shareSessionController);
+router.use('/', shareSessionController);
```

Comment:
What is the intention with that ?
Is that just for session, and then we'd have another one for QR ?

Reply:
Instead of creating routes inside `index.js` I thought of just making it use controllers.
> Is that just for session, and then we'd have another one for QR ?

Yes.